composer.json

```diff
@@ -90,7 +91,7 @@
     },
     "scripts": {
         "phpstan": [
-            "phpstan analyse lib tests"
+            "phpstan analyse lib tests --memory-limit 1G"
```

Comment:
There's possibly a lower appropriate value.

Reply:
That's fine - I guess it is just nice to have it "explode" gracefully if some analysis eats up memory, rather than locking up a whole machine when physical memory runs out.